DESCRIPTION

```diff
@@ -33,7 +33,8 @@ Suggests:
     purrr,
     rmarkdown,
     roxygen2,
-    testthat (>= 3.0.0)
+    testthat (>= 3.0.0),
+    usethis
```

Comment:
FYI I added usethis to `Suggests:`. We should probabbly do the same with readr and vroom (but first let's discuss if we need vroom or if readr is enough).